python/ray/_private/services.py

```diff
@@ -1343,6 +1343,7 @@ def start_raylet(redis_address,
                  redis_password=None,
                  metrics_agent_port=None,
                  metrics_export_port=None,
+                 tracing_startup_hook=None,
```

Comment:
we don't need this parameter in services.py anymore, it's not used here. (contrary to the one in parameter.py which is used)